circus-train-hive-view/src/main/java/com/hotels/bdp/circustrain/hive/view/transformation/TableProcessor.java

```diff
@@ -35,12 +35,15 @@
   @Override
   public Object process(Node node, Stack<Node> stack, NodeProcessorCtx procCtx, Object... nodeOutputs)
     throws SemanticException {
-    ASTNode astNode = (ASTNode) node;
+    ASTNode astNode;
```

Comment:
I prefer left, this is really satisfying checkstyle for the sake of it I think

Reply:
I agree, especially since we're throwing the exception that Java would throw anyway. I wonder if we should turn this one off? Generally if you're forcing a cast then you at least think you know what you're doing. What's the exact warning that checkstyle gives here?